libs/libjemalloc/Makefile

```diff
@@ -0,0 +1,52 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=jemalloc
+PKG_VERSION:=5.3.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/facebook/jemalloc/archive/refs/tags/
+PKG_SOURCE_URL_FILE:=$(PKG_VERSION).tar.gz
```

Comment:
Does this variable even exist?

```suggestion
```

Reply:
https://github.com/openwrt/openwrt/blob/main/include/download.mk#L322

Reply:
Weird. Maybe this time, it seems like my bug in my build env. Give me sec. I will try it once again.